### ISSUE TITLE
Add special TheBrain Key of ACRNM x Asus G14 (GA401IVC)

### ISSUE THREAD

**Issue by AntonSoeffing** [seerge/g-helper#1254]
Hey, I own the ACRNM x ASUS G14 also known as RMT-01 and it has a special key that is not yet identified. 

The button used to open a special software that came preinstalled. I would like to have the ability to configure it that way again.

I tried manually adding a M5, M6,... entry to the config file of ghelper. But that did not help :(.

![IMG_20230905_021405548](https://github.com/seerge/g-helper/assets/26146401/226697ef-c744-4f81-ad11-6e47f42ae351)

Could you please add this key, even though this laptop is limited to 3000 units - I would be very thankful as this is the only thing missing for me.

**Comment by squee666** [comment]
press the key with ghelper open

then go to your log file in %appdata% / Roaming / Ghelper
at the bottom of the file it should say what the Key pressed was

**Comment by seerge** [comment]
@AntonSoeffing hello, as other user have mentioned open 

``%appdata/ghelper/log.txt%`` and press that key, it will write down a keycode from it into the log

There is a good chance that key is already being processed by the app, just being called differently

**Comment by AntonSoeffing** [comment]
Thanks guys! I got the key now, its `55`! 
@seerge unfortunately I do not understand how to actually use this code besides switching profiles.

I want to map an .exe to it that opens when I press it. 

like this

` "m3_custom": "C:\\Users\\anton\\AppData\\Local\\Obsidian\\Obsidian.exe",`

But for the special key (55) instead of m3. How would I do that?

Thanks again guys for your work!

**Comment by seerge** [comment]
@AntonSoeffing try this build. I have called it M6

[GHelper.zip](https://github.com/seerge/g-helper/files/12526981/GHelper.zip)


I didn't add it to UI, but you can control it via config

```
"m6": "custom",
"m6_custom": "C:\\Users\\anton\\AppData\\Local\\Obsidian\\Obsidian.exe",
```

**Comment by AntonSoeffing** [comment]
Works like a charm!! Thank you so much! I guess I won't be able to update and have to stick to that build? That's alright I guess considering that everyting works now!

Thanks again!

**Comment by seerge** [comment]
It's already included in 0.119. Closing as completed
